dev/benchmarks/microbenchmarks/lib/geometry/matrix_utils_transform_rect_bench.dart

```diff
@@ -0,0 +1,139 @@
+  ];
+  final List<Offset> _offsets = <Offset>[
+    const Offset(1.1, 1.2),
+    const Offset(1.5, 1.8),
+    const Offset(0.0, 0.0),
+    const Offset(-1.1, -1.2),
+    const Offset(-1.5, -1.8),
+  ];
+
+  // Warm up lap
+  for (int i = 0; i < _kNumWarmUp; i += 1) {
+    final Rect rect = _rects[i % _rects.length];
+    final Offset offset = _offsets[i % _offsets.length];
+    final Matrix4 transform = (i > _kNumWarmUp / 2)
+        ? _perspectiveTransforms[i % _perspectiveTransforms.length]
+        : _transforms[i % _transforms.length];
+    MatrixUtils.transformRect(transform, rect);
+    MatrixUtils.transformPoint(transform, offset);
+  }
+
+  final Stopwatch watch = Stopwatch();
+  watch.start();
+  for (int i = 0; i < _kNumIterations; i += 1) {
+    final Rect rect = _rects[i % _rects.length];
+    final Matrix4 transform = _perspectiveTransforms[i % _perspectiveTransforms.length];
+    MatrixUtils.transformRect(transform, rect);
+  }
+  watch.stop();
+  final int rectMicrosecondsPerspective = watch.elapsedMicroseconds;
+
+  watch.reset();
+  watch.start();
+  for (int i = 0; i < _kNumIterations; i += 1) {
+    final Rect rect = _rects[i % _rects.length];
+    final Matrix4 transform = _transforms[i % _transforms.length];
+    MatrixUtils.transformRect(transform, rect);
+  }
+  watch.stop();
+  final int rectMicrosecondsAffine = watch.elapsedMicroseconds;
+
+  watch.reset();
+  watch.start();
+  for (int i = 0; i < _kNumIterations; i += 1) {
+    final Offset offset = _offsets[i % _offsets.length];
+    final Matrix4 transform = _perspectiveTransforms[i % _perspectiveTransforms.length];
+    MatrixUtils.transformPoint(transform, offset);
+  }
+  watch.stop();
+  final int pointMicrosecondsPerspective = watch.elapsedMicroseconds;
+
+  watch.reset();
+  watch.start();
+  for (int i = 0; i < _kNumIterations; i += 1) {
+    final Offset offset = _offsets[i % _offsets.length];
+    final Matrix4 transform = _transforms[i % _transforms.length];
+    MatrixUtils.transformPoint(transform, offset);
+  }
+  watch.stop();
+  final int pointMicrosecondsAffine = watch.elapsedMicroseconds;
+
+  final BenchmarkResultPrinter printer = BenchmarkResultPrinter();
+  const double scale = 1000.0 / _kNumIterations;
+  printer.addResult(
+    description: 'MatrixUtils.transformRectPerspective',
+    value: rectMicrosecondsPerspective * scale,
+    unit: 'ns per iteration',
+    name: 'MatrixUtils_persp_transformRect_iteration',
+  );
+  printer.addResult(
+    description: 'MatrixUtils.transformRectAffine',
+    value: rectMicrosecondsAffine * scale,
+    unit: 'ns per iteration',
+    name: 'MatrixUtils_affine_transformRect_iteration',
+  );
+  printer.addResult(
+    description: 'MatrixUtils.TransformPointPerspective',
+    value: pointMicrosecondsPerspective * scale,
+    unit: 'ns per iteration',
+    name: 'MatrixUtils_persp_transformPoint_iteration',
+  );
+  printer.addResult(
+    description: 'MatrixUtils.TransformPointAffine',
```

Comment:
For consistency with other names, let's not capitalize `TransformPointAffine`.

Reply:
See above.